### ISSUE TITLE
Absent central pair complex (9+0 pattern) at principal piece

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#5602]
**Preferred term label:**
feature in sperm flagella

**Synonyms**

**Definition (free text, please give PubMed ID)**

https://pubmed.ncbi.nlm.nih.gov/31278745/
**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
